FESS-GUI/conversionlib.h

```diff
@@ -0,0 +1,12 @@
+#ifndef CONVERSIONLIB_H
```

Comment:
I think conversions.h would be a better name for this file. "lib.h" is just a little redundant as well as inconsistent with our other file names